fv3core/stencils/a2b_ord4.py

```diff
@@ -22,485 +36,435 @@
 sd = utils.sd
 
 
-def grid():
-    return spec.grid
-
-
-@gtstencil()
-def ppm_volume_mean_x(qin: sd, qx: sd):
-    with computation(PARALLEL), interval(...):
-        qx[0, 0, 0] = b2 * (qin[-2, 0, 0] + qin[1, 0, 0]) + b1 * (qin[-1, 0, 0] + qin)
+@gtscript.function
+def ppm_volume_mean_x(qin: FloatField):
```

Comment:
Are we type-hinting the functions too? I realize I have reviewed and accepted other PRs you've opened with these haha. So no objection, just trying to remember when we changed our minds on that (previously we had suggested only type hinting stencils and some functions on a case-by-case basis). I haven't been adding type-hints to functions. We can probably do a style-consistency sweep at some point to pick up the stragglers.

Reply:
I've started adding them because once we have 2D arrays it will be somewhat necessary to keep track of things (although it should work with or without type hints). I thought it's confusing for the reader to see only 2 indices on some fields without the type hint telling the user that's really supposed to be a 2D field.